src/pages/Profile/index.jsx

```diff
@@ -65,9 +112,35 @@ function Profile() {
           });
       }
     );
-    setVisibile(false);
+    setVisible(false);
+  };
+
+<<<<<<< HEAD
```

Comment:
why is this here, isn't that causing syntax error? have you checked/tried running?

Reply:
Fixed